src/OSPSuite.InstallationValidator.Core/ApplicationConfiguration.cs

```diff
@@ -28,7 +27,7 @@ protected override string ApplicationFolderPathWithRevision(string revision)
       public override string ProductNameWithTrademark { get; }
       public override ApplicationIcon Icon { get; } = ApplicationIcons.Comparison;
       public override string UserSettingsFileName { get; }
-      public override string IssueTrackerUrl { get; }
+      public override string IssueTrackerUrl => Assets.Constants.Captions.IssueTrackerUrl;
```

Comment:
This could be updated as well to use a {get} as opposed to method. This will definitely not changed. Can be done in future PR